lib/associations/mixin.js

```diff
@@ -8,11 +8,12 @@ var Utils = require('./../utils')
   , BelongsTo = require('./belongs-to');
 
 /**
- * Creating assocations in sequelize is done by calling one of the belongsTo / hasOne / hasMany functions on a model (the source), and providing another model as the first argument to the function (the target).
+ * Creating assocations in sequelize is done by calling one of the belongsTo / hasOne / hasMany / belongsToMany functions on a model (the source), and providing another model as the first argument to the function (the target).
  *
- * * hasOne - adds a foreign key to target
- * * belongsTo - add a foreign key to source
- * * hasMany - adds a foreign key to target, unless you also specify that target hasMany source, in which case a junction table is created with sourceId and targetId
+ * * hasOne - adds a foreign key to the target and singular association mixins to the source.
+ * * belongsTo - add a foreign key and singular association mixins to the source.
+ * * hasMany - adds a foreign key to target and plural association mixins to the source.
+ * * belongsToMany - creates an N:M association with a join table and adds plural association mixins to both the source and the target. The junction table is created with sourceId and targetId.
```

Comment:
A single association still only adds the mixins to the source I believe - You can use a single if you only need to access it from one side, but for both sides you need two